modules/account-lib/CHANGELOG.md

```diff
@@ -3,6 +3,37 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [22.0.0](https://github.com/BitGo/BitGoJS/compare/@bitgo/account-lib@4.12.0...@bitgo/account-lib@22.0.0) (2024-02-19)
+
```

Comment:
Why this change log is from 4.12.0 -> 22.0.0. Shouldn't this be from 21.0.0->22.0.0. The breaking change which is mentioned below, I think that was done long back, and is the major version update done considering that breaking change?

Reply:
This is generated automatically, and is probably missing the right tag for version 21.